README.md

```diff
@@ -23,6 +23,8 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * **Name**: Name of this DHCP Scope
 * **SubnetMask**: Subnet mask for the scope specified in IP address format
 * **LeaseDuration**: Time interval for which an IP address should be leased
+ * This should be specified in the following format: `Days.Hours:Minutes:Seconds`
```

Comment:
@TravisEz13 Not sure why we need a helper method? In hindsight, it should have just been implemented as an integer, specifying the number of minutes. We could have then leveraged the ConvertTo-TimeSpan/ConvertFrom-TimeSpan helper methods in the xActiveDirectory module.